docs/sphinx/source/contributing.rst

```diff
@@ -15,9 +19,13 @@ Here are a few ideas for you can contribute, even if you are new to
 pvlib-python, git, or Python:
 
 * Make `GitHub issues <https://github.com/pvlib/pvlib-python/issues>`_
-  and contribute to the conversation about how to resolve them.
+  and contribute to the conversations about how to resolve them.
```

Comment:
In the broader sense, answering questions on stackoverflow and/or discussing on the google group would be contributions as well.

Reply:
yes, absolutely!